statistics-reports.md

```diff
@@ -0,0 +1,202 @@
+# Displaying usage statistics
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Statistics for a DSpaceObject
+**GET /api/statistics/usage-reports**
+
+As we don't have a use case to iterate over all the existing usage reports, the main endpoint is not implemented and a 405 error code is returned according to our [general error response codes](README.md#Error-codes).
+
+## Single statistic
+**GET /api/statistics/usage-reports/<:id>**
+
+This endpoint provides a specific statistic
+
+An example JSON response document to `/api/statistics/usage-reports/1911e8a4-6939-490c-b58b-a5d70f8d91fb_TopCountries`:
+```json
+{
+    "id": "1911e8a4-6939-490c-b58b-a5d70f8d91fb_TopCountries",
+    "type": "usage-report",
+    "report-type": "TotalVisits",
```

Comment:
which are the possible values and meaning for this attribute?

Reply:
Although there's no theoretical limit to the types of reports, they match the various tables on the item/collection/community/site statistics pages.
I've created a list of the values which currently exist, and updated the examples to match the IDs